code/help.py

```diff
@@ -472,17 +603,145 @@ def help_alphabet(ab: dict):
         #         gui_context_help.showing
         #     )
         # )
-        gui_context_help.hide()
-        gui_alphabet.hide()
+        hide_all_help_guis()
         gui_alphabet.show()
         register_events(False)
         actions.mode.enable("user.help")
 
+    def help_number_key_words(ab: dict):
```

Comment:
You could probably merge all these 'help_\*' and 'gui_\*' functions/imgui instances into a single action since you're passing in the help contents anyway.